### DISCUSSION TITLE
How to set no data text in react-table

### DISCUSSION THREAD

**Discussion by SergGolub89** [TanStack/table#2969] [Q&A]
Hi. 
Starting to work with this library, but didn't find any info in the documentation.
I'm not importing a full table from react-table, it needs to be custom(used a basic example from "Get Started" and then improved according to needs of the project).
Tried to overwrite ReactTableDefaults(found decision in google), but it's not working.
For now, if the data variable is empty, the table also is empty, but I need in this case row with 'Nothing found' string inside.
Can someone share with experience how to solve this problem?
Thank you in advance.

**Comment by rahulmathews** [comment]
You can use something like this:
```
Cell: row => {<br>
        if(row.value === ''){
   return <div> Nothing Found </div>
}
else{
  return <div> row.value <div>
}
      },
```
